concepticondata/conceptlists/Bowern-2021-342b.tsv

```diff
@@ -0,0 +1,343 @@
+Bowern-2021-342b-90	90	coot	28	flora-fauna	Flora Fauna Vocabulary		
+Bowern-2021-342b-91	91	cotamundi, coatimundi	156	flora-fauna	Flora Fauna Vocabulary	2389	COATI (COATIMUNDI)
+Bowern-2021-342b-92	92	cotton	157	flora-fauna	Flora Fauna Vocabulary	1850	COTTON
+Bowern-2021-342b-93	93	cottontail rabbit	29	flora-fauna	Flora Fauna Vocabulary		
+Bowern-2021-342b-94	94	cottonwood	30	flora-fauna	Flora Fauna Vocabulary		
+Bowern-2021-342b-95	95	coyote	31	flora-fauna	Flora Fauna Vocabulary	912	COYOTE
+Bowern-2021-342b-96	96	crab	158	flora-fauna	Flora Fauna Vocabulary	2348	CRAB
+Bowern-2021-342b-97	97	crested pigeon	271	flora-fauna	Flora Fauna Vocabulary		
+Bowern-2021-342b-98	98	cricket	159	flora-fauna	Flora Fauna Vocabulary	2354	CRICKET
+Bowern-2021-342b-99	99	crow	32	flora-fauna	Flora Fauna Vocabulary	913	CROW
+Bowern-2021-342b-100	100	cunuri (tree)	231	flora-fauna	Flora Fauna Vocabulary		
+Bowern-2021-342b-101	101	curassow	160	flora-fauna	Flora Fauna Vocabulary	315	CURASSOW
+Bowern-2021-342b-102	102	darter/cormorant	272	flora-fauna	Flora Fauna Vocabulary	840	CORMORANT
+Bowern-2021-342b-103	103	datura	33	flora-fauna	Flora Fauna Vocabulary		
+Bowern-2021-342b-104	104	death adder, five-minute snake	273	flora-fauna	Flora Fauna Vocabulary		
+Bowern-2021-342b-105	105	deer	161	flora-fauna	Flora Fauna Vocabulary	1936	DEER
+Bowern-2021-342b-106	106	deer, mule	34	flora-fauna	Flora Fauna Vocabulary		
+Bowern-2021-342b-107	107	digger pine	35	flora-fauna	Flora Fauna Vocabulary		
+Bowern-2021-342b-108	108	dingo	274	flora-fauna	Flora Fauna Vocabulary	2460	DINGO
+Bowern-2021-342b-109	109	dog (camp, domestic)	36	flora-fauna	Flora Fauna Vocabulary	2009	DOG
+Bowern-2021-342b-110	110	dog (wild; bush dog)	162	flora-fauna	Flora Fauna Vocabulary	2448	WILD DOG
+Bowern-2021-342b-111	111	dogbane	37	flora-fauna	Flora Fauna Vocabulary		
+Bowern-2021-342b-112	112	dove	38	flora-fauna	Flora Fauna Vocabulary	1853	DOVE
+Bowern-2021-342b-113	113	duck	39	flora-fauna	Flora Fauna Vocabulary	1360	DUCK
+Bowern-2021-342b-114	114	dugong	275	flora-fauna	Flora Fauna Vocabulary		
+Bowern-2021-342b-115	115	earthworm	276	flora-fauna	Flora Fauna Vocabulary	2350	EARTHWORM
+Bowern-2021-342b-116	116	edible palm-dwelling larva (palm weevil)	163	flora-fauna	Flora Fauna Vocabulary		
+Bowern-2021-342b-117	117	elder	40	flora-fauna	Flora Fauna Vocabulary		
+Bowern-2021-342b-118	118	electric eel	164	flora-fauna	Flora Fauna Vocabulary	2443	ELECTRIC EEL
+Bowern-2021-342b-119	119	elk	41	flora-fauna	Flora Fauna Vocabulary	1772	ELK
+Bowern-2021-342b-120	120	emu	277	flora-fauna	Flora Fauna Vocabulary	2817	EMU
+Bowern-2021-342b-121	121	firefly, lightning bug	185	flora-fauna	Flora Fauna Vocabulary	206	FIREFLY
+Bowern-2021-342b-122	122	fish (generic)	42	flora-fauna	Flora Fauna Vocabulary	227	FISH
+Bowern-2021-342b-123	123	fish (large type)	165	flora-fauna	Flora Fauna Vocabulary		
+Bowern-2021-342b-124	124	fish poison, barbasco	166	flora-fauna	Flora Fauna Vocabulary	348	FISH POISON
```

Comment:
This seems to be a specific poison from the plant. Because the concept is already mapped to a more generic gloss in another list, I'd unmap it here.